### PR TITLE
Allow blank assignees when editing ticket

### LINKED ISSUE #1529: Ticket admin should allow blank assignee
Otherwise editing a ticket without assignee will force you to assign someone

### PR DESCRIPTION
**Pull request by zecookiez**
Fixes #1529

### CODE REVIEW

**Comment by TheAvidDev** [comment]
I think you have to generate a migration when you change this.

**Review by TheAvidDev — APPROVED** [review]
LGTM, edit your PR body to include `Fixes #1529` so GitHub recognizes it.

File: judge/migrations/0111_blank_assignees_ticket.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0110_default_output_prefix_override'),
+    ]
+
+    operations = [
+         migrations.AlterField(
```
Comment (kiritofeng) [inline judge/migrations/0111_blank_assignees_ticket.py:13]: Indentation is wrong here, did you try to edit this migration manually?

**Review by kiritofeng — CHANGES_REQUESTED** [review]

File: judge/migrations/0111_blank_assignees_ticket.py (hunk 1 above)
Comment (zecookiez) [reply judge/migrations/0111_blank_assignees_ticket.py]: I did edit this manually, and accidentally left an extra space 🤦‍♂️

Should be fixed now.

**Review by kiritofeng — APPROVED** [review]
LGTM

**Pull request merged**